### DISCUSSION TITLE
How does Cobbler 3.2 support traditional mode installation

### DISCUSSION THREAD

**Discussion by QinQiang666** [cobbler/cobbler#3512] [Q&A]
How to configure and support traditional mode installation when installing the system in Cobbler 3.2 bios (Legacy) mode and getting an IP address through dhcp?

dhcp configuration
![image](https://github.com/cobbler/cobbler/assets/29726716/fcbd5d88-c4b2-4821-8036-c7995cbffbb6)

It's stuck here
![image](https://github.com/cobbler/cobbler/assets/29726716/bf9db0e3-8252-4359-b9a3-f2a827a0dc6c)

**Comment by SchoolGuy** [comment]
So essentially we would need a switch that tells Cobbler not to append `inst.ks`/`kickstart`/`autoyast`/... to the append line. You can temporarily remove those, but Cobbler will keep regenerating the TFTP config.

Regarding the screen you are stuck at: There is no error message visible and it looks like the usual workflow. Could it be that something is timing out due to a firewall that is closed?
